adhocracy-plus/assets/scss/components/_button.scss

```diff
@@ -4,103 +4,136 @@
     font-family: inherit;
     font-size: inherit;
     margin-bottom: 0;
+    text-decoration: none;
 }
 
 .btn {
     @extend %button-base;
-
-    text-decoration: none;
 }
 
-.btn--light {
-    @extend .btn-outline-light;
+.btn.btn--primary {
```

Comment:
Why do these all have the btn and the variant? To make it more specific and overwrite other styles?

Reply:
@fuzzylogic2000 it was to make it more specific so we could still specify styling for just the `btn`class but it actually didn't work as planned, and i'm not sure if we should have anything more then basic styling assigned to btn anyway and as that was the black btn and given that it is difficult to show different states i'm not sure what to do for that one, once this is merged i'll let Marcelo know and we are going to have a chat about it tomorrow

Reply:
Cool, let's merge then!